src/Persistence/Event/LifecycleEventArgs.php

```diff
@@ -41,6 +42,13 @@ public function __construct($object, ObjectManager $objectManager)
      */
     public function getEntity()
     {
+        Deprecation::trigger(
+            'doctrine/persistence',
+            'https://github.com/doctrine/persistence/pull/281',
```

Comment:
Let's refer to the original PR or issue instead?

Reply:
I wanted to, but simple annotation led me to code migration 🙈 I'll dig further

Reply:
yeaaah, deprecated for 10 years now! https://github.com/doctrine/persistence/commit/aca6f031881f71e79f081ebd2a5ae7913d731d33

Reply:
The PR is https://github.com/doctrine/common/pull/222

Reply:
Oh right, there was a split!